src/configs/ManuscriptsEditor.ts

```diff
@@ -71,6 +71,7 @@ export interface EditorProps {
   ) => void
   lockBody: boolean
   isViewingMode?: boolean
+  hiddenNodeTypes?: string[] | undefined
```

Comment:
I feel `excludedNodeTypes` or `blockedNodeTypes` might be clearer than `hiddenNodeTypes`.